doc/09-object-types.md

```diff
@@ -742,6 +742,19 @@ to evenly distribute all checks over a certain period of time, i.e. to avoid loa
 > The `address` and `address6` attributes are required for running commands using
 > the `$address$` and `$address6$` runtime macros.
 
+> **Info**
+>
+> The macros $host.name$, $service.description$, $host.address$ or $address$ are available for forming the notes url and action url
```

Comment:
The macros should be written with code tags e.g. `$host.name$`

Reply:
service.description is not a valid macro name in Icinga 2. I don't understand this specific info note either, as you can use more than these. This sounds more like a specific macro parser in Icinga Web 2. Such an advanced merger should be avoided in the basic Icinga 2 docs.

Reply:
I would also prefer to have the Icinga Web 2 macros used instead of the limited set of compatibility ones. So it would be `$service_description$` instead of `$service.description$` and `$host_name$` instead of `$host.name$`, the same for other internal attributes and e.g. `$_host_os$` for custom variable `host.vars.os`. I wanted to point to the corresponding Icinga Web 2 documentation but found https://github.com/Icinga/icingaweb2/issues/2302 still open.